www/js/controller.js

```diff
@@ -865,7 +865,7 @@ define([
 				console.log("Parser error: ", err);
 				console.log(err.message);
 				console.log(err.Error);
-				this.setAuthorStatus(this.currentID,'equation','incorrect');
+				this._model.authored.setAuthorStatus(this.currentID,'equation','incorrect');
```

Comment:
There are 3 instances of this and we can use this._mode == AUTHOR check and use ".active" ? we can still make a card as a check like you said when we further made con-student and stabilize the whole controller